webviz_subsurface/plugins/_tornado_plotter_fmu/_plugin.py

```diff
@@ -0,0 +1,277 @@
+                self._ensemble_name,
+            ),
+            PlugInIDs.SharedSettings.FILTERS,
+        )
+
+        # Settingsgroup for the view options
+        self.add_store(
+            PlugInIDs.Stores.ViewSetttings.REFERENCE,
+            WebvizPluginABC.StorageType.SESSION,
+        )
+        self.add_store(
+            PlugInIDs.Stores.ViewSetttings.SCALE, WebvizPluginABC.StorageType.SESSION
+        )
+        self.add_store(
+            PlugInIDs.Stores.ViewSetttings.SENSITIVITIES,
+            WebvizPluginABC.StorageType.SESSION,
+        )
+        self.add_store(
+            PlugInIDs.Stores.ViewSetttings.RESET,
+            WebvizPluginABC.StorageType.SESSION,
+        )
+        self.add_store(
+            PlugInIDs.Stores.ViewSetttings.PLOT_OPTIONS,
+            WebvizPluginABC.StorageType.SESSION,
+        )
+        self.add_store(
+            PlugInIDs.Stores.ViewSetttings.LABEL, WebvizPluginABC.StorageType.SESSION
+        )
+        self.add_shared_settings_group(
+            ViewSettings(design_matrix_df), PlugInIDs.SharedSettings.VIEW_SETTINGS
+        )
+
+        # Stores for data
+        self.add_store(
+            PlugInIDs.Stores.DataStores.TORNADO_DATA,
+            WebvizPluginABC.StorageType.SESSION,
+        )
+        self.add_store(
+            PlugInIDs.Stores.DataStores.CLICK_DATA, WebvizPluginABC.StorageType.SESSION
+        )
+        self.add_store(
+            PlugInIDs.Stores.DataStores.HIGH_LOW, WebvizPluginABC.StorageType.SESSION
+        )
+        self.add_store(
+            PlugInIDs.Stores.DataStores.CLIENT_HIGH_PIXELS,
+            WebvizPluginABC.StorageType.SESSION,
+        )
+
+        self.add_view(
+            self._tornado_widget,
+            PlugInIDs.TornardoPlotGroup.TORNADO_PLOT,
+            PlugInIDs.TornardoPlotGroup.GROUPNAME,
+        )
+
+    @property
+    def tour_steps(self) -> List[dict]:
+        """Tour of the plugin"""
+        return [
+            {
+                "id": self.view(PlugInIDs.TornardoPlotGroup.TORNADO_PLOT)
+                .layout_element(TornadoWidget.IDs.MAIN_COLUMN)
+                .get_unique_id(),
+                "content": ("Shows tornado plot."),
+            },
+            {
+                "id": self.shared_settings_group(
+                    PlugInIDs.SharedSettings.PLOTPICKER
+                ).component_unique_id(PlotPicker.IDs.BARS_OR_TABLE),
+                "content": "Choose between showing the data with bars og in a table",
+            },
+            {
+                "id": self.shared_settings_group(
+                    PlugInIDs.SharedSettings.SELECTORS
+                ).component_unique_id(Selectors.IDs.RESPONSE),
+                "content": "Choose the respose for the data",
+            },
+            {
+                "id": self.shared_settings_group(
+                    PlugInIDs.SharedSettings.FILTERS
+                ).component_unique_id(Filters.IDs.SINGLE_FILTER),
```

Comment:
The tour is not working for me. It seems you are using dictionaries as IDs for the multi filter components (REGION and ZONE) because you're having an unknown number of selectors. Since the tour steps are static you cannot react to these dynamic elements. Hence, it would make more sense to create a div container around your selectors and giving that one an ID that you then use in the tour steps.

Reply:
The tour steps is not working for me either. I have done tours completely similar on other plugins which works perfectly fine, and it also worked previously on this plugin so I don't understand why it is not working right now